database/migrations/2022_10_17_000000_default_uuid_field.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration {
```

Comment:
Why is this necessary in this PR?

Reply:
I was unsure on whether to include this myself, I decided to do it because I could not get the Twitter code to "fail properly" without it